lib/trento/hosts/host.ex

```diff
@@ -507,6 +524,39 @@ defmodule Trento.Hosts.Host do
     |> Multi.execute(&maybe_emit_host_health_changed_event/1)
   end
 
+  def execute(
+        %Host{host_id: host_id},
+        %CompleteSoftwareUpdatesDiscovery{
+          host_id: host_id,
+          relevant_patches: %RelevantPatches{
+            total: total,
+            security_advisories: security_advisories,
+            bug_fixes: bug_fixes,
+            software_enhancements: software_enhancements
+          }
+        }
+      )
+      when security_advisories + bug_fixes + software_enhancements != total,
+      do: {:error, :inconsistent_software_updates_discovery}
```

Comment:
Is this something that we care about? I would get rid of this clause, I'm pretty sure that the total you're getting is the sum of the lengths of all the lists :smile:

Reply:
well, I was doubtful myself, I could just get rid of the total. I was just thinking of making it extremely simple for someone consuming that possibily projected/broadcasted information to not have to do sums, bot i can live wiothout that